tests/Feature/Api/V1Test.php

```diff
@@ -255,7 +255,7 @@ public function testGetAchievementUnlocks(): void
         /** @var Game $game */
         $game = Game::factory()->create(['ConsoleID' => $system->ID]);
         /** @var Achievement $achievement */
-        $achievement = Achievement::factory()->create(['GameID' => $game->ID]);
+        $achievement = Achievement::factory()->create(['GameID' => $game->ID, 'Flags' => AchievementType::OfficialCore]);
```

Comment:
two more places where `published()` should be used.

Reply:
Yep, good catch. Addressed in https://github.com/RetroAchievements/RAWeb/pull/1530/commits/11fa9a2f079927caf2caf08407b924813a66754c